layer.yaml

```diff
@@ -1,2 +1,6 @@
 includes:
   - 'layer:basic'
+options:
+    basic:
+        use_venv: true
```

Comment:
Because of this, it seems like the removal in acitons.py makes sense.  From a code perspective, this all makes total sense.